packages/components/src/components/button-link-text-switch/types.ts

```diff
@@ -0,0 +1,10 @@
+import { Generic } from '@a11y-ui/core';
+import { ButtonOrLinkOrTextWithChildrenProps } from '../../components';
+
+type RequiredProps = {
+	prop: ButtonOrLinkOrTextWithChildrenProps;
```

Comment:
keine neuen Properties einführen!

Reply:
Wie jetzt?

Reply:
Prop ist ein neues Property. Nimmer konsistenterweise `_links`.

Reply:
porp in link umbenennen